packages/core-api/__tests__/v2/handlers/transactions.test.js

```diff
@@ -168,7 +168,6 @@ describe('API 2.0 - Transactions', () => {
         expect(response.data.data).toBeArray()
 
         expect(response.data.data).toHaveLength(100)
-        expect(response.data.meta.totalCount).toBe(153)
```

Comment:
Notice: that count is supposed to be an estimate. So we may return 100 rows, totalCount=100, but however there may be e.g. 110 rows in the database.